src/pineko/cli/theory_.py

```diff
@@ -32,11 +32,11 @@ def opcards(theory_id, datasets, overwrite):
 @theory_.command()
 @click.argument("theory_id", type=click.INT)
 @click.argument("datasets", type=click.STRING, nargs=-1)
-@click.option("--logs", is_flag=True, help="dump logs")
+@click.option("--no-logs", is_flag=True, help="suppress logs")
```

Comment:
Here and below: if you put directly `--no-logs` follow the pattern on the [docs](https://click.palletsprojects.com/en/8.0.x/options/#boolean-flags) and provide both, and set the default to `False`.

Reply:
`True` you mean? because you said logs by default